src/main/kotlin/com/hedvig/rapio/qa/QaController.kt

```diff
@@ -0,0 +1,28 @@
+package com.hedvig.rapio.qa
+
+import com.hedvig.rapio.qa.dto.UnsignMemberRequest
+import com.hedvig.rapio.qa.dto.UnsignMemberResponse
+import org.springframework.context.annotation.Profile
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@Profile("staging", "development")
+@RequestMapping("v1/staging/")
+class QaController(
+    private val qaService: QaService
+) {
+    @PostMapping("unsignMember")
+    fun unsignMember(
+        @RequestBody request: UnsignMemberRequest
+    ): ResponseEntity<UnsignMemberResponse> = ResponseEntity.ok(
+        UnsignMemberResponse(
+            qaService.unsignMember(
+                personalNumber = request.personalNumber
+            )
+        )
+    )
```

Comment:
I personally like "noContent" if the result is not interesting. 

```suggestion
    ): ResponseEntity<Void> {
        val success = qaService.unsignMember(personalNumber = request.personalNumber)
        if (success) {
          return ResponseEntity.noContent().build()
        } else {
          return ResponseEntity.badGateway().build()
        }
    }
```

Reply:
I like that, but the error should probably be notFound